src/utils.ts

```diff
@@ -9,3 +9,14 @@ export function debounce(fn, wait = 0) {
         }, wait);
     };
 }
+
+export const minZoom = 3;
+
+export const backgroundLayer = {
+    url: "https://server.arcgisonline.com/ArcGIS/rest/services/Canvas/World_Light_Gray_Base/MapServer/tile/{z}/{y}/{x}",
+    attribution:
+        "Tiles &copy; Esri &mdash; Esri, DeLorme, NAVTEQ; " +
+        "Boundaries: <a href='https://gadm.org' target='_blank'>GADM</a> version 4.1",
+    maxZoom: 10,
+    minZoom
+};
```

Comment:
I get that it makes the Choro module a bit tidier to pull these out, but since they're only ever going to be used to Chorpleth, wouldn't mind them going into `composables/utils` or even `choropleth/map/utils` then could potentially tidy Legend and ResetMapButton into the `map` folder too.

Reply:
i prefer the idea of `choropleth/map/utils` but gone with `composables/utils` for now, typescript complaining about `computed` not being found, etc, not actually sure what the problem is, i skimmed over the tsconfig and vite config files, would rather not mess around with lots of config options :laughing: ticket is here: https://mrc-ide.myjetbrains.com/youtrack/agiles/103-67/current?issue=mrc-5351

Reply:
Sorry, I completely mistyped there, I meant `components/utils` or `components/map/utils`! :facepalm:
Shouldn't be in composables!